src/App/ChaosRecipeEnhancer.UI/UserControls/SettingsForms/GeneralForms/GeneralForm.xaml

```diff
@@ -251,7 +251,8 @@
             <ComboBoxItem Content="Portuguese" />
             <ComboBoxItem Content="Russian" />
             <ComboBoxItem Content="Spanish" />
-            <ComboBoxItem Content="Chinese" />
```

Comment:
Looks good! Thanks for your help with this, we definitely need to review some of the language/translation stuff later. Without your help I would have never known (as I am clearly not a native speaker) :) 

We have plans to support the Korean and other clients as well, this is a great step in that direction.